src/main/java/me/zeroeightsix/kami/module/modules/movement/AutoWalk.kt

```diff
@@ -26,7 +32,9 @@ import net.minecraftforge.fml.common.network.FMLNetworkEvent
 )
 class AutoWalk : Module() {
     @JvmField
-    var mode: Setting<AutoWalkMode> = register(Settings.e("Direction", AutoWalkMode.BARITONE))
+    var mode: Setting<AutoWalkMode> = register(Settings.e("Direction", AutoWalkMode.PATHFIND))
+    val flydelay = register(Settings.integerBuilder("SableFlightDelay").withRange(0, 400).withValue(0).withVisibility { mode.value == AutoWalkMode.PATHFIND }.build())
+    private val drawPath = register(Settings.booleanBuilder("DrawPath").withValue(false).withVisibility { mode.value == AutoWalkMode.PATHFIND }.build())
```

Comment:
1. shouldn't this be on by default?
2. are you still using the bleh GL code or did you switch to Baritone's?